module/item/item-sheet.js

```diff
@@ -341,8 +341,8 @@ export class SWSEItemSheet extends ItemSheet {
 
         let ownedItem = actor.items.get(droppedItem.data._id);
 
-        let isItemMod = Object.values(droppedItem.data.data.attributes).find(attr => attr.key === "itemMod");
-        if (isItemMod?.value === "true") {
+        let isItemMod = (droppedItem.data.type === "upgrade") ? "true" : "false";
```

Comment:
this is incorrect.  different item types can be used as upgrades in this system and use the itemMod attribute to handle that